In [1]:
from sympy import *

## Sample Problem 1 - Total Change from a Table

In [4]:
year = [1960, 1970, 1980, 1990, 2000, 2010]
rate = [36, 40, 63, 64, 10, 8]
n = 5

In [6]:
left_rect = [rate[i]*(year[i+1]-year[i]) for i in range(0,n)]
LHS = sum(left_rect)
LHS

2130

**a)** The total change between 1960 and 2010 was approximately 2,130,000 people.

**b)** The function is neither increasing nor decreasing, therefore it cannot be determined whether the approximation is an overestimate or an underestimate.

## Sample Problem 2 - Interpreting the Definite Integral

**a)** The definite integrate from $a$ to $b$ represents the total value the machine has lost (in thousands of dollars) in the time delta $b - a$. 

**b)**

In [10]:
t = symbols('t')

r = 24*exp(-0.22*t)
r

24*exp(-0.22*t)

In [13]:
integrate(r, (t, 0, 5))

72.7776999602095

In [14]:
200_000 - (integrate(r, (t, 0, 5)).evalf() * 1000)

127222.300039790

The value of the machine 5 years after it was purchased is $127222.30003979

## Sample Problem 3 - Integrals and Area

In [15]:
x = symbols('x')

h = 16 - x**2
h

16 - x**2

**a)**

In [16]:
integrate(h, (x, 0, 6))

24

**b)**

In [17]:
solveset(h, x)

{-4, 4}

In [18]:
# 4 is inside our interval [0, 6]

# area above x axis
above = integrate(h, (x, 0, 4))
# area below x axis
below = -integrate(h, (x, 4, 6))
# total area
total_area = above + below
total_area.evalf()

61.3333333333333